### PR TITLE
Avoiding black bars

### PR DESCRIPTION
**Pull request by wmathurin**
Went back to original email with launch images etc
Renamed images to have their actual sizes
Re-assigned launch images
Seems to avoid black bars on all targets

### CODE REVIEW

File: shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json [hunk 1]
```diff
@@ -1,52 +1,46 @@
...
     },
     {
       "orientation" : "portrait",
-      "idiom" : "ipad",
+      "idiom" : "iphone",
+      "filename" : "Default640x960.png",
       "extent" : "full-screen",
-      "minimum-system-version" : "8.0",
-      "filename" : "Default-Portrait@2x.png",
+      "minimum-system-version" : "7.0",
```
Comment (bhariharan) [inline shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json]: Should be ```8.0```.

File: shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json [hunk 2]
```diff
@@ -1,52 +1,46 @@
...
     {
-      "orientation" : "landscape",
-      "idiom" : "ipad",
       "extent" : "full-screen",
-      "minimum-system-version" : "8.0",
-      "filename" : "Default-Landscape@2x.png",
+      "idiom" : "iphone",
+      "subtype" : "retina4",
+      "filename" : "Default640x1136.png",
+      "minimum-system-version" : "7.0",
```
Comment (bhariharan) [inline shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json]: Should be ```8.0```.

**Comment by bhariharan** [comment]
Looks good otherwise.

**Comment by khawkins** [comment]
:+1:

File: shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json (hunk 2 above)
Comment (wmathurin) [inline shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json]: XCode made the change.
When I manually change it to 7.0, I get a compiler warning:
"(...)/SalesforceMobileSDK-iOS/shared/resources/Images.xcassets: An iPhone Retina (4-inch) launch image for iOS 7.0 and later is required."
Then you get the black bar for those devices.

Comment (bhariharan) [inline shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json]: Sounds good.

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**